fem/bilininteg.hpp

```diff
@@ -2136,24 +2136,72 @@ class VectorMassIntegrator: public BilinearFormIntegrator
    const GeometricFactors *geom;  ///< Not owned
    int dim, ne, nq, dofs1D, quad1D;
 
+#ifdef MFEM_USE_CEED
+   // CEED extension
+   CeedData* ceedDataPtr;
+#endif
```

Comment:
Some of these `#ifdef` blocks could be removed if `CeedData*` is defined as `void*` when `MFEM_USE_CEED` is not defined. It might make the code a little bit less noisy.

Reply:
I think I have addressed this, can you check if you like it?